ptsprojects/zephyr/gap.py

```diff
@@ -71,6 +71,10 @@ class CHAR:
 
 iut_device_name = 'Tester'
 iut_manufacturer_data = 'ABCD'
+iut_appearance = '1111'
+iut_svc_data = '1111'
+iut_flags = '11'
+iut_svcs = '1111'
```

Comment:
This value reflects the service that is actually in GATT database. Maybe it's good to change this value to e.g. GAP service UUID